mir_eval/segment.py

```diff
@@ -958,6 +961,10 @@ def nce(reference_intervals, reference_labels, estimated_intervals,
         beta for F-measure
         (Default value = 1.0)
 
+    marginal : bool
+        If `False`, normalize conditional entropy by uniform entropy.
+        If `True`, normalize conditional entropy by the marginal entropy.
+
     Returns
     -------
     S_over
```

Comment:
Should we change the math in this part of the docstring to note that it can be different according to the value of `marginal`?  For example denominator should be `H(y_ref)` right?  Or at least, we should change them in the `v_measure` docstring right?